packages/SwingSetRunner/bin/runner

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
```

Comment:
let's name the new directory `swingset-runner`.. I kept the uppercased `SwingSet` when I assembled the monorepo from the earlier individual repos, but our modern convention is all-lowercase package names.